packages/postcss-svgo/src/__tests__/index.js

```diff
@@ -198,6 +198,15 @@ test('should warn on SVG containing unclosed tags', async () => {
   expect(result.messages[0].type).toBe('warning');
 });
 
+test('should only warn with svg data uri', async () => {
+  const css = `@font-face {
+  src: url("https://example/dfds.woff2") format("woff2"),
```

Comment:
Something weird here. The test fails with an error inside `postcss-value-parser` if I try a relative URL (`url(example/dfds.woff2)`). I assume `url()` handling has been tested elsewhere?